### DISCUSSION TITLE
Using 0th hour vs 1st hour for historical HRRR forecasts together with current forecast

### DISCUSSION THREAD

**Discussion by rsargent** [NOAA-GSL/GSL-Regional-Model-Forum#20] [Q&A] [answered]
We use the arlftp archive of past HRRR forecasts (ftp.arl.noaa.gov/archives/hrrr/) for modeling per-hour air pollution exposures from major emitters, and it's been incredibly useful for this application.

We are moving to building real-time models that start from hours before time and end with hours after the current time, and have a question about historical data.  We're wondering if we ought to use the 0th hour or 1st hour of historical forecasts when we combine them with current forecasts?  We note that the arlftp archive moved from 0th hour to 1st hour in late 2020, and are curious what motivated the change to 1st hour given what I imagine to be the larger uncertainty further in the future.

To make the question more concrete, if we're assembling HRRR hours for a HYSPLIT run that starts in hours before current time and continues through future hours, our thought is to start from the 0th-18th hours from the most current HRRR hourly forecast, and to add before that the hours from previous forecasts.  So for example if the most current forecast starts at 1200 and we're building HRRR hours 1000-1400, we're considering these two sequences of HRRR sources:

Selecting 0th hour for historical forecasts:
  10:00 from 0th hour of past forecast starting 10:00
  11:00 from 0th hour of past forecast starting 11:00
  12:00 from 0th hour of current forecast starting 12:00
  13:00 from 1st hour of current forecast starting 12:00
  14:00 from 2nd hour of current forecast starting 12:00

Selecting 1st hour for historical forecasts:
  10:00 from 1st hour of past forecast starting 09:00
  11:00 from 1st hour of past forecast starting 10:00
  12:00 from 0th hour of current forecast starting 12:00
  13:00 from 1st hour of current forecast starting 12:00
  14:00 from 2nd hour of current forecast starting 12:00

Thanks for thoughts on 0th vs 1st hour, and any wisdom you might have about our best approach here.

-- Randy
Randy Sargent
Carnegie Mellon CREATE Lab

**Comment by EricJames-NOAA** [comment, marked as answer]
Hi Randy, thanks for your question!  We're glad to hear that the HRRR works well for your HYSPLIT application.   

I'm not familiar with the ARL archive, but in general the 0-h fields should be more accurate in terms of wind, due to the assimilation of recent observations.   So we would recommend using the 0-h fields wherever possible.  

If you have trouble finding 0-h files, NCEP provides realtime HRRR data here:

https://www.nco.ncep.noaa.gov/pmb/products/hrrr/

And there are several archives of historical HRRR data available online (see more details here: 

https://rapidrefresh.noaa.gov/hrrr/

**Reply by rsargent** [reply]
Thanks for the quick reply!
